ur_description/config/ur10/physical_parameters.yaml

```diff
@@ -0,0 +1,77 @@
+# Physical parameters
+
+dh_parameters:
+  d1: 0.1273
+  a2: -0.612
+  a3: -0.5723
+  d4: 0.163941
+  d5: 0.1157
+  d6: 0.0922
+
+offsets:
+  shoulder_offset: 0.220941 # measured from model
+  elbow_offset: -0.1719 # measured from model
+  upper_arm_inertia_offset: 0.045 # measured from model
+
+inertia_parameters:
+  base_mass: 4.0  # This mass might be incorrect
+  shoulder_mass: 7.778
+  upper_arm_mass: 12.93
+  forearm_mass: 3.87
+  wrist_1_mass: 1.96
+  wrist_2_mass: 1.96
+  wrist_3_mass: 0.202
+
+  shoulder_radius: x0.060   # FROM UR5 DON'T USE
```

Comment:
We don't use these anywhere, so we could also drop them, right? This could go for all models, I think.